cognite_toolkit/_cdf_tk/builders/_base.py

```diff
@@ -178,8 +187,16 @@ def build(
                 if warning is not None:
                     yield [warning]
                 continue
-            destination_path = self._create_destination_path(source_file.source.path, loader.kind)
 
+            if validation == "identifier":
```

Comment:
This seems to be duplicated in many builders. What about just making an `xIdentifierBuilder`? and use that if the validation in `identifier`? 

This also seems to be duplicating the `.check_built_resource` that is in the build command. Why do you need to do this here at all?